### ISSUE TITLE
strconv.ParseFloat: parsing "TRK UT EX CAP M1152A1": invalid syntax

### ISSUE THREAD

**Issue by PowerTap** [tealeg/xlsx#104]
I'm using the library to parse an XLSX document, every time I try to get the String value of a cell I get the error:

strconv.ParseFloat: parsing "TRK UT EX CAP M1152A1": invalid syntax

Clearly the internal Value is text, and I'm using the .String() method, so I'm not sure why the library is trying to parse the data as a Float.  

If I pull the value directly with .Value rather than .String() I get the correct value, so I'm not sure exactly what is happening with the case statement in FormattedValue() that is making the library think that it is a Float rather than text.  

Is it because the cell has formatted text?

Thanks,
Andy

**Comment by qiuly** [comment]
I also encountered this problem today, and after 1+ hours debugging, I found the reason finally! haha!
The error was locate at Line 202 in cell.go file.  

195 func (c *Cell) FormattedValue() string {
196     var numberFormat string = c.GetNumberFormat()
197     switch numberFormat {
198     case "general":
199         return c.Value
200     case "0", "#,##0":
201         return c.formatToInt("%d")
202     case "0.00", "#,##0.00", "@":
203         return c.formatToFloat("%.2f")

the format "@" means c.Value should be displayed as text, but here the code are trying to convert it to float. 

so the solution is as follows:(when the format is "@", deal it just the same as "genenal")
195 func (c *Cell) FormattedValue() string {
196     var numberFormat string = c.GetNumberFormat()
197     switch numberFormat {
198     case "general", "@":
199         return c.Value
200     case "0", "#,##0":
201         return c.formatToInt("%d")
202     case "0.00", "#,##0.00":
203         return c.formatToFloat("%.2f")

I've already tested it, it works!
